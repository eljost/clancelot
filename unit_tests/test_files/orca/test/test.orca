! HF-3c MORead
%maxcore 1000
%moinp "previous.gbw"
*xyz 0 1
H 0.000000 0.000000 0.000000 
H 1.000000 0.000000 0.000000 
*
