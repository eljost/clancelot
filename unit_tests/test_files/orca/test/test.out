
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #  Department of molecular theory and spectroscopy    #
                  #              Directorship: Frank Neese              #
                  # Max Planck Institute for Chemical Energy Conversion #
                  #                  D-45470 Muelheim/Ruhr              #
                  #                       Germany                       #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 3.0.3 - RELEASE   -


 With contributions from (in alphabetic order):
   Ute Becker             : Parallelization
   Dmytro Bykov           : SCF Hessian
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Dimitrios Liakos       : Extrapolation schemes; parallel MDCI
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA and ECA modules, normal mode analysis, Resonance Raman, ABS, FL, XAS/XES, NRVS
   Christoph Reimann      : Effective Core Potentials
   Michael Roemelt        : Restricted open shell CIS
   Christoph Riplinger    : Improved optimizer, TS searches, QM/MM, DLPNO-CCSD
   Barbara Sandhoefer     : DKH picture change effects
   Igor Schapiro          : Molecular dynamics
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse, T. Risthaus : VdW corrections, initial TS optimization,
                                                   DFT functionals, gCP
   Ed Valeev                                     : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, R. Olivares           : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Andreas Klamt, Michael Diedenhofen            : otool_cosmo (COSMO solvation model)
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


Your calculation utilizes the basis: MINIX
by S. Grimme

Your calculation utilizes the geometrical counterpoise correction gCP
Please cite in your paper:
H.Kruse, S. Grimme J.Chem.Phys., 136, (2012), 154101 
   

Your calculation utilizes the atom-pairwise dispersion correction
with the Becke-Johnson damping scheme (D3BJ)
Cite in your paper:
S.Grimme, S.Ehrlich, L.Goerigk, J Comput Chem, (2011), 32, 1456â€“1465 
S.Grimme, J.Antony, S.Ehrlich and H.Krieg, J.Chem.Phys., 132, (2010), 154104 
   

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================
Now building the actual basis set


INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = test.orca
|  1> ! HF-3c MORead
|  2> %maxcore 40
|  3> %moinp "previous.gbw"
|  4> *xyz 0 1
|  5> H 0.000000 0.000000 0.000000
|  6> H 1.000000 0.000000 0.000000
|  7> *
|  8> 
|  9>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  H      0.000000    0.000000    0.000000
  H      1.000000    0.000000    0.000000

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG    MASS        X           Y           Z
   0 H     1.0000    0     1.008          0.000000000000000          0.000000000000000          0.000000000000000
   1 H     1.0000    0     1.008          1.889726133921252          0.000000000000000          0.000000000000000
* core charge reduced due to ECP

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 H      0   0   0   0.000000     0.000     0.000
 H      1   0   0   1.000000     0.000     0.000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 H      0   0   0   0.000000     0.000     0.000
 H      1   0   0   1.889726     0.000     0.000

---------------------
BASIS SET INFORMATION
---------------------
There are 1 groups of distinct atoms

 Group   1 Type H   : 3s contracted to 1s pattern {3}

Atom   0H    basis set group =>   1
Atom   1H    basis set group =>   1

-------------------------
ECP PARAMETER INFORMATION
-------------------------




------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------

                         BASIS SET STATISTICS AND STARTUP INFO

 # of primitive gaussian shells          ...    6
 # of primitive gaussian functions       ...    6
 # of contracted shell                   ...    2
 # of contracted basis functions         ...    2
 Highest angular momentum                ...    0
 Maximum contraction depth               ...    3
 Integral package used                   ... LIBINT
 Integral threshhold            Thresh   ...  1.000e-10
 Primitive cut-off              TCut     ...  1.000e-11
 Storage cut-off                TCutStore...  1.000e-10
 Buffer length                           ... 1048576 integrals/buffer
 Generate file with integrals 4 distinct ...false


                              INTEGRAL EVALUATION

 One electron integrals    
   ECP integrals                                       ... done
 Pre-screening matrix                    ... done
 Shell pair data                         ... done (   0.000 sec)
 Beginning two electron integral evaluation:
Initializing the integral list in ORCA_GTOInt
 Starting repulsion integrals
  <ss|ss>  ...            6 b.         0 skpd (  0.0%)     0.000 s. (  0.003 ms/b)


                                      INTEGRAL STATISTICS

Integral type statistics:
 Number of integrals retained            ...          6 
 One   Center Integrals retained         ...          6 (100.0%)
 Two   Center Integrals retained         ...          0 (  0.0%)
 Three Center Integrals retained         ...          0 (  0.0%)
 Four  Center Integrals retained         ...          0 (  0.0%)

Storage statistics:
 Number of char integrals                ...          0 (  0.0%)
 Number of short int integrals           ...          0 (  0.0%)
 Number of double/float integrals        ...          6 (100.0%)
 Amount of memory used                   ...          0 MB

Prescreening statistics:
 Total number of shell quadruples        ...          6 
 Number of skipped shell quadruples      ...          0 (  0.0%)
 # of negligible batches done            ...          0 (  0.0%)

Timings:
 Total evaluation time               ...     0.079 sec (   0.001 min)
 One electron matrix time            ...     0.004 sec (   0.000 min) =  4.9%
 Schwartz matrix evaluation time     ...     0.042 sec (   0.001 min) = 53.2%
 Repulsion integral time             ...     0.000 sec (   0.000 min) =  0.0%

-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... test.orca
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....    2
 Basis Dimension        Dim             ....    2
 Nuclear Repulsion      ENuc            ....      0.5291772083 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Conventional
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh
 Integral Buffer length BufferLength    .... 1048576
 Integral index format  IndFormat       ....     0
 Integral value format  ValFormat       ....     0
 Integral Storage       Thresh          ....  1.000e-10 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 5.111e-01
Time for diagonalization                   ...    0.000 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.003 sec
Total time needed                          ...    0.003 sec

-----------------------------------------
INITIAL GUESS: POLARIZED ATOMIC DENSITIES
-----------------------------------------
The minimal basis has 2 functions
EHT matrix was diagonalized
Basis set overlap was calculated now projecting the orbitals
Using GuessMode=FMatrix for the initial guess
C*T matrix was formed
Effective Fockian was calculated
Effective Fockian was diagonalized
Initial density was built
                      ------------------
                      INITIAL GUESS DONE
                      ------------------

 InCore treatment chosen:
   Memory dedicated               ...     40 MB
   Memory needed                  ...      0 MB
   Number of tiny    integrals    ...       0
   Number of small   integrals    ...       0
   Number of regular integrals    ...       6

--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
                      *** Initiating the SOSCF procedure ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  0     -1.06366770  -1.0636676984  0.000000  0.000000  0.000000  0.000000
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   1 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :           -1.06366770 Eh             -28.94387 eV

Components:
Nuclear Repulsion  :            0.52917721 Eh              14.39964 eV
Electronic Energy  :           -1.59284491 Eh             -43.34351 eV

One Electron Energy:           -2.22656166 Eh             -60.58782 eV
Two Electron Energy:            0.63371676 Eh              17.24431 eV

Virial components:
Potential Energy   :           -2.26044088 Eh             -61.50972 eV
Kinetic Energy     :            1.19677318 Eh              32.56585 eV
Virial Ratio       :            1.88877969


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...    0.0000e+00  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    0.0000e+00  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    0.0000e+00  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    4.6618e-17  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    0.0000e+00  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (test.orca.gbw) ****
             **** DENSITY FILE WAS UPDATED (test.orca.scfp.tmp) ****
             **** ENERGY FILE WAS UPDATED (test.orca.en.tmp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000      -0.479564       -13.0496 
   1   0.0000       0.458952        12.4887 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 H :    0.000000
   1 H :    0.000000
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 H s       :     1.000000  s :     1.000000
  1 H s       :     1.000000  s :     1.000000


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 H :    0.000000
   1 H :    0.000000

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 H s       :     1.000000  s :     1.000000
  1 H s       :     1.000000  s :     1.000000


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 H      1.0000     1.0000     0.0000     1.0000     1.0000     0.0000
  1 H      1.0000     1.0000     0.0000     1.0000     1.0000     0.0000

  Mayer bond orders larger than 0.1
B(  0-H ,  1-H ) :   1.0000 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.025 sec
Sum of individual times     ....       0.014 sec  ( 57.3%)

Fock matrix formation       ....       0.006 sec  ( 24.2%)
Diagonalization             ....       0.000 sec  (  1.1%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.007 sec  ( 27.1%)
Initial guess               ....       0.000 sec  (  0.9%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.0%)
SOSCF solution              ....       0.001 sec  (  4.0%)

Entering the calculation of the dispersion correction...

Checking the input atom types...

Done


-------------------------------------------------------------------------------
                          DFT DISPERSION CORRECTION                            
                                                                               
                                DFTD3 V2.1  Rev 6                               
                          USING Becke-Johnson damping                          
-------------------------------------------------------------------------------
Hartree-Fock is recognized
Active option DFTDOPT                   ...         4   

molecular C6(AA) [au] = 28.931678 


            DFT-D V3
 parameters
 using HF/MINIX parameters
 s6 scaling factor         :     1.0000 
 a1 scaling factor         :     0.4171 
 s8 scaling factor         :     0.8777 
 a2 scaling factor         :     2.9149 
 ad hoc parameters k1-k3   :    16.0000     1.3333    -4.0000 

 Edisp/kcal,au: -1.016715727343  -0.001620239472 
 E6   /kcal   :  -0.651819683 
 E8   /kcal   :  -0.364896045 
 % E8         :  35.889682324 

-------------------------   ----------------
Dispersion correction           -0.001620239
-------------------------   ----------------

Egcp:       0.0020492904 a.u.
Ebas:      -0.0006525351 a.u.
------------------   -----------------
gCP+bas correction         0.001396755
------------------   -----------------
----------------------------   ----------------
Total correction to HF/MINIX       -0.000223484
----------------------------   ----------------


-------------------------   --------------------
FINAL SINGLE POINT ENERGY        -1.063891182618
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... test.orca.gbw
Electron density file                           ... test.orca.scfp.tmp

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:     -0.00000      -0.00000      -0.00000
Nuclear contribution   :      0.00000       0.00000       0.00000
                        -----------------------------------------
Total Dipole Moment    :      0.00000       0.00000       0.00000
                        -----------------------------------------
Magnitude (a.u.)       :      0.00000
Magnitude (Debye)      :      0.00000


Timings for individual modules:

Sum of individual times         ...        0.162 sec (=   0.003 min)
GTO integral calculation        ...        0.116 sec (=   0.002 min)  71.1 %
SCF iterations                  ...        0.047 sec (=   0.001 min)  28.9 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 0 seconds 420 msec
